dashboard/app/models/levels/level.rb

```diff
@@ -727,6 +727,20 @@ def localized_rubric_property(property)
     end
   end
 
+  def localized_validations
+    if should_localize?
+      validations.each do |validation|
```

Comment:
I suggest cloning the validations and editing the copy first.
In general, we want to preserve the original data, specifically, it can avoid side effects if validations are used somewhere else in the code. 

Here's an example:
https://github.com/code-dot-org/code-dot-org/blob/ad90d0f431b63944da72a5ff3a2ad701d3b1aed3/dashboard/app/models/activity_section.rb#L65-L73

Reply:
Done!